app/MMB/src/scenes/list/cityImage/StatusBadge.js

```diff
@@ -14,8 +13,8 @@ type Props = {|
 |};
 
 const StatusBadge = (props: Props) => {
-  const status = idx(props.data, _ => _.status) || '';
-  const isPastBooking = idx(props.data, _ => _.isPastBooking) || '';
+  const status = props.data.status ?? '';
+  const isPastBooking = props.data.isPastBooking ?? false;
```

Comment:
Above, we used `Boolean` to wrap that data I believe. Would it be necessary here too?

Reply:
Above we used `Boolean(something?.isPastBooking)` I believe

It achieves kind of the same thing as ?? false

Reply:
Changed it to using Boolean as well